<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpDownWidget</class>
 <widget class="QWidget" name="UpDownWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="WidgetMain" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#WidgetMain
{
background: transparent;
border-left:0px;
border-top:0px;
border-right:0px;
border-bottom:1px solid rgb(111,111,111);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LabelName">
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(80, 181, 72);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelAction">
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: rgb(61, 61, 61);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PButtonIgnore">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
font: 12px &quot;宋体&quot;;
border:1px solid rgb(135,135,135);
border-radius:3px;
color: rgb(135, 135, 135);</string>
        </property>
        <property name="text">
         <string>忽略</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PButtonAgree">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
font: 12px &quot;宋体&quot;;
border:1px solid rgb(255,60,26);
border-radius:3px;
color: rgb(255,60,26);</string>
        </property>
        <property name="text">
         <string>同意</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
